release-notes/opensearch-security.release-notes-2.2.0.0.md

```diff
@@ -0,0 +1,14 @@
+## 2022-08-11 Version 2.2.0.0
+
+Compatible with OpenSearch 2.2.0
+
+### Test Fix
+
+* Abstract waitForInit to minimize duplication and improve test reliability ([#1935](https://github.com/opensearch-project/security/pull/1935))
+* Create a manually started workflow for bulk run of integration tests ([#1937](https://github.com/opensearch-project/security/pull/1937))
+
```

Comment:
These commits are missing:
[Tenant info action test ](https://github.com/opensearch-project/security/pull/1935)

[Create a manually started workflow for bulk run of integration tests](https://github.com/opensearch-project/security/pull/1937)

Reply:
Those commits are included. 

- https://github.com/opensearch-project/security/commit/7dcf342eb8043308e7480c8238129c83e4cba84f
- https://github.com/opensearch-project/security/commit/297b3431bb01c8d86de9ec76599262b813fe41d9

Reply:
yep I see it..